joint_deconvolution/joint_deconvolution_and_feather_auto.py

```diff
@@ -0,0 +1,99 @@
+import json, os, sys, glob
+
+region      = sys.argv[1]
+pth         = os.path.abspath(os.getcwd())
+
+with open(f"./default_tclean_commands.json", "r") as fh:
+    tclean_commands = json.load(fh)
+
+os.chdir(pth+'/'+region)
+if not os.path.exists('clean_12m_7m'):
+    os.mkdir('clean_12m_7m')
+os.chdir(pth+'/'+region+'/clean_12m_7m/')
+
+vis_all     = glob.glob(pth+'/'+region+'/twelve_m/spws/*.spw3') + glob.glob(pth+'/'+region+'/seven_m/spws/*.spw3')
```

Comment:
general recommendation (python-wide): either do
```suggestion
vis_all     = glob.glob(os.path.join(pth, region, '/twelve_m/spws/*.spw3')) + glob.glob(os.path.join(region, '/seven_m/spws/*.spw3'))
```
or
`f'{pth}/{region}/twelve/...`

The former is more robust (it works on windows, for example), while the latter is easier to read